src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -117,6 +121,10 @@ public function trans($message, array $arguments = [], $domain = null, $locale =
             $arguments['%count%'] = $count;
         }
 
+        if (null === $this->translator && !interface_exists(TranslatorInterface::class)) {
```

Comment:
I don't get this: you cannot have the interface not-exist and have the trait exist at the same time (they're both provided by the same package)